### PR TITLE
Define Element base class

### LINKED ISSUE #7: Define Element inteface

### PR DESCRIPTION
**Pull request by pavelanihimovsky**
resolves #7

### CODE REVIEW

File: aquality_selenium_core/elements/Element.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    @property
+    def element(self, timeout: timedelta = None) -> WebElement:
+        """
+        Gets current element by specified locator.
+        Default timeout is provided in TimeoutConfiguration.
+        Throws NoSuchElementException if element not found.
+        :return: Instance of WebElement if found.
+        """
+        raise NotImplementedError
```
Comment (a-hramovich) [inline aquality_selenium_core/elements/Element.py]: Better to add @abstractmethod annotation instead error raise. Error will be raised by application if child doesn't implement abstract method.

File: aquality_selenium_core/elements/Element.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+    def click(self) -> None:
+        """
+        Clicks on the item.
+        """
+        raise NotImplementedError
+
+    @property
+    def _element_state(self):
```
Comment (a-hramovich) [inline aquality_selenium_core/elements/Element.py]: The same method (and methods below) has public signature in java\C# . 
Do we need protected in python for them?

File: aquality_selenium_core/elements/Element.py (hunk 1 above)
Comment (pavelanihimovsky) [reply aquality_selenium_core/elements/Element.py]: This method won't be abstract. I've added exception till it is implemented.

File: aquality_selenium_core/elements/Element.py (hunk 2 above)
Comment (pavelanihimovsky) [reply aquality_selenium_core/elements/Element.py]: Are you sure that these methods are public in Java/C#. All of them are protected.

**Pull request closed without merging**